hibernate-jpamodelgen is causing problems

**Describe the bug**
I started a the 'Quarkus - Get Started'-project (See https://quarkus.io/get-started/) in order to have an initial code base.
After that I added JPA and tried to integrate 'hibernate-jpamodelgen' in order to use the 'Metamodel Generator'. 
Adding the hibernate-jpamodelgen was done via:
```
<artifactId>maven-compiler-plugin</artifactId>
<version>3.8.1</version>
<configuration>
	<annotationProcessorPaths>
		<annotationProcessorPath>
			<groupId>org.hibernate</groupId>
			<artifactId>hibernate-jpamodelgen</artifactId>
			<version>5.4.27.Final</version>
		</annotationProcessorPath>
		<annotationProcessorPath>
			<groupId>io.quarkus</groupId>
			<artifactId>quarkus-panache-common</artifactId>
			<version>${quarkus.platform.version}</version>
		</annotationProcessorPath>
	</annotationProcessorPaths>
</configuration>
</plugin>
```
As described in https://quarkus.io/guides/hibernate-orm-panache#defining-entities-in-external-projects-or-jars.

'mvn clean compile quarkus:dev' leads to the following error:
```
[ERROR] /home/jonas/private-workspace/getting-started/target/generated-sources/annotations/org/acme/getting/started/Person_.java:[10,71] cannot find symbol
  symbol:   class PanacheEntity_
  location: package io.quarkus.hibernate.orm.panache
```

**Expected behavior**
'mvn clean compile quarkus:dev' works and the 'Metamodel Generator' works as well

**Actual behavior**

'mvn clean compile quarkus:dev' leads to the following error:
```
[ERROR] /home/jonas/private-workspace/getting-started/target/generated-sources/annotations/org/acme/getting/started/Person_.java:[10,71] cannot find symbol
  symbol:   class PanacheEntity_
  location: package io.quarkus.hibernate.orm.panache
```


**To Reproduce**

Steps to reproduce the behavior:
1. `git checkout git@github.com:the-bug/hibernate-jpamodelgen-reproducer.git`
2. `cd hibernate-jpamodelgen-reproducer`
3. `mvn clean compile quarkus:dev`

**Configuration**
-

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Linux ... 5.4.0-65-generic #73-Ubuntu SMP Mon Jan 18 17:25:17 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: 
 ```
openjdk version "11.0.9.1" 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04)
OpenJDK 64-Bit Server VM (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04, mixed mode, sharing)
```
 - GraalVM version (if different from Java): not used 
 - Quarkus version or git rev: 1.11.1.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
 ```
Maven home: /home/.../.m2/wrapper/dists/apache-maven-3.6.3-bin/1iopthnavndlasol9gbrbg6bf2/apache-maven-3.6.3
Java version: 11.0.9.1, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.4.0-65-generic", arch: "amd64", family: "unix"
```

**Additional context**
(Add any other context about the problem here.)

--- a/extensions/panache/hibernate-orm-panache/deployment/src/test/java/io/quarkus/hibernate/orm/panache/deployment/test/PanacheEntityMetaModelTest.java
+++ b/extensions/panache/hibernate-orm-panache/deployment/src/test/java/io/quarkus/hibernate/orm/panache/deployment/test/PanacheEntityMetaModelTest.java
@@ -1,0 +1,13 @@
+package io.quarkus.hibernate.orm.panache.deployment.test;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntity_;
+
+public class PanacheEntityMetaModelTest {
+    @Test
+    public void testMetaModelExistence() {
+        Assertions.assertEquals("id", PanacheEntity_.ID);
+    }
+}

--- a/extensions/panache/hibernate-orm-panache/runtime/pom.xml
+++ b/extensions/panache/hibernate-orm-panache/runtime/pom.xml
@@ -85,7 +85,9 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <compilerArgument>-proc:none</compilerArgument>
+                    <annotationProcessors>
+                        <annotationProcessor>org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor</annotationProcessor>
+                    </annotationProcessors>
                 </configuration>
             </plugin>
             <plugin>

--- a/extensions/panache/hibernate-reactive-panache/deployment/src/test/java/io/quarkus/hibernate/reactive/panache/test/PanacheEntityMetaModelTest.java
+++ b/extensions/panache/hibernate-reactive-panache/deployment/src/test/java/io/quarkus/hibernate/reactive/panache/test/PanacheEntityMetaModelTest.java
@@ -1,0 +1,13 @@
+package io.quarkus.hibernate.reactive.panache.test;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.hibernate.reactive.panache.PanacheEntity_;
+
+public class PanacheEntityMetaModelTest {
+    @Test
+    public void testMetaModelExistence() {
+        Assertions.assertEquals("id", PanacheEntity_.ID);
+    }
+}

--- a/extensions/panache/hibernate-reactive-panache/runtime/pom.xml
+++ b/extensions/panache/hibernate-reactive-panache/runtime/pom.xml
@@ -89,7 +89,9 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <compilerArgument>-proc:none</compilerArgument>
+                    <annotationProcessors>
+                        <annotationProcessor>org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor</annotationProcessor>
+                    </annotationProcessors>
                 </configuration>
             </plugin>
             <plugin>

